drivers/vcnl40x0/include/vcnl40x0_internals.h

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright (C) 2017 Inria
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     drivers_vcnl40x0
+ * @{
+ *
+ * @file
+ * @brief       Internal addresses, registers, constants for the VCNL40X0 devices.
+ *
+ * @author      Alexandre Abadie <alexandre.abadie@inria.fr>
+ * @}
+ */
+
+#ifndef VCNL40X0_INTERNALS_H
+#define VCNL40X0_INTERNALS_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @name VCNL40X0 I2C address
+ * @{
+ */
+#define VCNL40X0_ADDR                                (0x13)
+#define VCNL40X0_PRODUCT_ID                          (0x20)
```

Comment:
this should be documented separately, or not? Its different from the I2C address.